src/tools/msc.lua

```diff
@@ -118,6 +118,10 @@
 		},
 		justmycode = {
 			Off = "false"
```

Comment:
I wrote `justmycode "On"` and got an error in the past... I think we should always apply `On` and `Off` as a whole to users.

Reply:
Correct.  I have that on my backlog, or feel free to add that in yourself in a separate PR :)

Reply:
Appreciate you pulling that off my backlog then